packages/@sanity/cli/src/studioDependencies.ts

```diff
@@ -4,6 +4,9 @@ export const studioDependencies = {
     // Official studio dependencies
     sanity: '^3.0.0',
 
+    // Official studio plugin dependencies
+    '@sanity/vision': '^3.0.0',
```

Comment:
This makes it so it ends up in all the templates?

Reply:
Yes, and that is what we want!